packages/tool_util/setup.cfg

```diff
@@ -36,8 +36,10 @@ install_requires =
     galaxy-util>=22.1
     conda-package-streaming
     lxml!=4.2.2
+    numpy
```

Comment:
numpy is a scary dependency and not required to parse Galaxy XML, etc.. I think we should move these to a new ``extra_require`` - maybe ``extended-assertions``.